pkg/controller/catalog/storage/mem_storage.go

```diff
@@ -16,211 +16,45 @@ limitations under the License.
 
 package storage
 
-import (
-	"fmt"
-	"log"
-	"strings"
-
-	model "github.com/kubernetes-incubator/service-catalog/model/service_controller"
-)
-
-type bindingPair struct {
-	binding    *model.ServiceBinding
-	credential *model.Credential
-}
-
 type memStorage struct {
-	brokers map[string]*model.ServiceBroker
-	// This gets fetched when a SB is created (or possibly later when refetched).
-	// It's static for now to keep compatibility, seems like this could be more dynamic.
-	catalogs map[string]*model.Catalog
-	// maps instance ID to instance
-	services map[string]*model.ServiceInstance
-	// maps binding ID to binding
-	// TODO: support looking up all bindings for a service instance.
-	bindings map[string]*bindingPair
+	brokers        *memStorageBroker
```

Comment:
The naming debate is definitely a good one to have, but IMO out of the scope here. This code was modified as little as possible to remove the package in the PR title.

Go convention would be a different package structure as was implied above, so maybe create an issue to revisit or do as part of #197?